From 5bf9baf3691b92023c63ae83957529c4c9f35039 Mon Sep 17 00:00:00 2001
From: Xin Ouyang <Xin.Ouyang@windriver.com>
Date: Thu, 5 Apr 2012 13:46:00 +0800
Subject: [PATCH 4/4] refpolicy-rbac: Define SELinux users.

Define the SELinux user, and bind them with the assigned roles and
MCS catetories.

Upstream-Status: Inappropriate [no upstream]
---
 policy/users |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/policy/users b/policy/users
index c4ebc7e..62d3897 100644
--- a/policy/users
+++ b/policy/users
@@ -43,3 +43,8 @@ ifdef(`direct_sysadm_daemon',`
 ',`
 	gen_user(root, sysadm, sysadm_r staff_r ifdef(`enable_mls',`secadm_r auditadm_r'), s0, s0 - mls_systemhigh, mcs_allcats)
 ')
+
+gen_user(engineer_u, engineer, engineer_r, s0, s0 - mcs_systemhigh, mcs_allcats)
+gen_user(senior_engineer_u, senior_engineer, senior_engineer_r engineer_r, s0, s0 - mcs_systemhigh, mcs_allcats)
+gen_user(manager_u, manager, manager_r senior_engineer_r engineer_r, s0, s0 - mcs_systemhigh, mcs_allcats)
+gen_user(technical_lead_u, technical_lead, technical_lead_r senior_engineer_r engineer_r, s0, s0 - mcs_systemhigh, mcs_allcats)
-- 
1.7.5.4

